File: Windows/NetworkCode/Account.cpp
#include "Account.h"

int Account::next_id = 0;

Account::Account(ServiceType serviceType)
{
	id = next_id;
	next_id++;
	this->serviceType = serviceType;
}

Account::~Account()
{
}

int Account::GetId()
{
	return id;
}

ServiceType Account::GetServiceType()
{
	return serviceType;
}

File: Windows/WindowsConsoleClient/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsConsoleClient
{
    class Account
    {
        private int id;
        private int serviceType;
        private string address;

        public Account(int id, int serviceType, string address)
        {
            this.id = id;
            this.serviceType = serviceType;
            this.address = address;
        }

        public int GetId()
        {
            return id;
        }

        public int GetServiceType()
        {
            return serviceType;
        }

        public string GetAddress()
        {
            return address;
        }
    }
}
